fprime_fpp/fprime_fpp_install.py

```diff
@@ -199,6 +199,7 @@ def install_fpp_via_git(installation_directory: Path):
     Should FPP not be available as a published version, this will clone the FPP repo, checkout, and build the FPP tools
     for the given version. This requires the following tools to exist on the system: git, sh, java, and sbt. These tools
     will be checked and then the process will run and intall into the specified directory.
+    Additionally installs the FPL (https://github.com/fprime-community/fprime-layout) tools unless FPL_NO_INSTALL is set to TRUE.
```

Comment:
Can we change the flag name from FPP_NO_INSTALL to FPP_INSTALL? In general, double-negative flags of the form X_IS_NOT_TRUE = TRUE are confusing and should be avoided. You could do the installation unless FPP_INSTALL = FALSE.